core/pkg/resourcehandler/filesloader.go

```diff
@@ -37,19 +37,62 @@ func (fileHandler *FileResourceHandler) GetResources(sessionObj *cautils.OPASess
 	// map resources based on framework required resources: map["/group/version/kind"][]<k8s workloads ids>
 	k8sResources := setK8sResourceMap(sessionObj.Policies)
 	allResources := map[string]workloadinterface.IMetadata{}
-	workloadIDToSource := make(map[string]reporthandling.Source, 0)
 	ksResources := &cautils.KSResources{}
 
-	workloads := []workloadinterface.IMetadata{}
-
 	if len(fileHandler.inputPatterns) == 0 {
 		return nil, nil, nil, fmt.Errorf("missing input")
 	}
-	path := fileHandler.inputPatterns[0]
+
+	logger.L().Info("Accessing local objects")
+	cautils.StartSpinner()
+
+	for path := range fileHandler.inputPatterns {
+		workloadIDToSource, workloads, err := getResourcesFromPath(fileHandler.inputPatterns[path])
+		if err != nil {
+			return nil, allResources, nil, err
+		}
+		if len(workloads) == 0 {
+			logger.L().Debug("path ignored because contains only a non-kubernetes file", helpers.String("path", fileHandler.inputPatterns[path]))
+		}
+
+		for k, v := range workloadIDToSource {
+			sessionObj.ResourceSource[k] = v
+		}
+
+		// map all resources: map["/apiVersion/version/kind"][]<k8s workloads>
+		mappedResources := mapResources(workloads)
+
+		// save only relevant resources
+		for i := range mappedResources {
+			if _, ok := (*k8sResources)[i]; ok {
+				ids := []string{}
+				for j := range mappedResources[i] {
+					ids = append(ids, mappedResources[i][j].GetID())
+					allResources[mappedResources[i][j].GetID()] = mappedResources[i][j]
+				}
+				(*k8sResources)[i] = append((*k8sResources)[i], ids...)
+			}
+		}
+
+	}
+
+	if err := fileHandler.registryAdaptors.collectImagesVulnerabilities(k8sResources, allResources, ksResources); err != nil {
+		logger.L().Warning("failed to collect images vulnerabilities", helpers.Error(err))
+	}
+
+	cautils.StopSpinner()
+	logger.L().Success("Accessed to local objects")
```

Comment:
typo: Accessed to local objects → Done accessing local objects

```suggestion
	logger.L().Success("Done accessing local objects")
```